### PR TITLE
add exhibitor service

### PR DESCRIPTION
**Pull request by cfortier2**

### CODE REVIEW

File: v1/fleet_units/exhibitor.service [hunk 1]
```diff
@@ -0,0 +1,25 @@
+[Unit]
+Description=Exhibitor/Zookeeper
+After=docker.service
+Requires=docker.service
+
+[Service]
+Environment=EXHIBITOR_IMAGE=mbabineau/zookeeper-exhibitor:3.4.6_1.5.4
```
Comment (ko-be) [inline v1/fleet_units/exhibitor.service]: Latest is 1.5.5 @cfortier2

File: v1/fleet_units/exhibitor.service [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+RestartSec=20
+TimeoutStartSec=0
+ExecStartPre=-/usr/bin/docker kill zookeeper-exhibitor
+ExecStartPre=-/usr/bin/docker rm zookeeper-exhibitor
+ExecStartPre=/usr/bin/docker pull ${EXHIBITOR_IMAGE}
+ExecStart=/usr/bin/docker run \
+          --net=host \
+          --name=zookeeper-exhibitor \
+          -e S3_BUCKET=`etcdctl get /zookeeper/s3_bucket` \
+          -e S3_PREFIX=`etcdctl get /zookeeper/s3_prefix` \
```
Comment (ko-be) [inline v1/fleet_units/exhibitor.service]: According to this https://registry.hub.docker.com/u/mbabineau/zookeeper-exhibitor/  it also needs -e HOSTNAME=blah as well

File: v1/fleet_units/exhibitor.service (hunk 1 above)
Comment (cfortier2) [inline v1/fleet_units/exhibitor.service]: they haven't built that in master yet:
image

**Comment by nBerg** [comment]
:+1:

**Pull request merged**